department-of-veterans-affairs/caseflow
# Allow n/a for judge non-availability

Connects #6529 

### Description
This PR allows the hearing coordinators to upload 'N/A' for a judge's nonavailability days.

### Testing Plan
1. Upload a spreadsheet with 'N/A' listed for a judge
1. Confirm it uploads with no errors
1. Confirm the judge has an entry in NonAvailabilities with a null date
1. Confirm algorithm runs successfully
